src/ucs/datastruct/ptr_array.c

```diff
@@ -26,17 +26,17 @@ static inline int ucs_ptr_array_is_free(ucs_ptr_array_t *ptr_array, unsigned ind
             __ucs_ptr_array_is_free(ptr_array->start[index]);
 }
 
-static inline uint32_t ucs_ptr_array_placeholder_get(ucs_ptr_array_elem_t elem)
+static inline uint32_t ucs_ptr_array_size_free_get(ucs_ptr_array_elem_t elem)
 {
     ucs_assert(__ucs_ptr_array_is_free(elem));
-    return elem >> UCS_PTR_ARRAY_PLCHDR_SHIFT;
+    return elem >> UCS_PTR_ARRAY_SIZE_FREE_SHIFT;
 }
 
-static inline void ucs_ptr_array_placeholder_set(ucs_ptr_array_elem_t *elem,
-                                                 uint32_t placeholder)
+static inline void UCS_F_MAYBE_UNUSED
+ucs_ptr_array_size_free_set(ucs_ptr_array_elem_t *elem, uint32_t size_free)
```

Comment:
where is it supposed to be used?

Reply:
Well, it's no longer used, I removed it.